hera_mc/cm_handling.py

```diff
@@ -318,7 +319,8 @@ def get_specific_connection(self, cobj, at_date=None):
         fnd = []
         for conn in self.session.query(partconn.Connections).filter(
                 (func.upper(partconn.Connections.upstream_part) == cobj.upstream_part.upper())
-                & (func.upper(partconn.Connections.downstream_part) == cobj.downstream_part.upper())):
+                & (func.upper(partconn.Connections.downstream_part)
+                   == cobj.downstream_part.upper())):
```

Comment:
marking another

Reply:
flake8 won't let me do this one either. I can't figure out why.

Reply:
Oh! I know why it worked on the earlier one -- because we have a `# noqa` on that line because sql requires `==` no `is`.

Reply:
Given that I can't do it on the others, I'm not going to do it on the earlier ones. You're welcome to try and see if you can come up with something aesthetically pleasing to you that flake8 will accept.

Reply:
That's fine :smile: I'm not super adamant about it anyway.

Reply:
update: I figured out how to do what flake8 and you wanted for this one and the above ones, but not for the ones below.